Index: src/Main/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Main;\r\n\r\nimport presentation.MenuHandler;\r\nimport java.util.Scanner;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        Scanner scanner = new Scanner(System.in);\r\n\r\n\r\n        // Inicializar el manejador de menú\r\n        presentation.MenuHandler menuHandler = new presentation.MenuHandler(scanner);\r\n\r\n        // Puedes pasar las instancias necesarias a las clases correspondientes\r\n\r\n        // Iniciar la aplicación\r\n        menuHandler.start();\r\n\r\n        scanner.close();\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main/Main.java b/src/Main/Main.java
--- a/src/Main/Main.java	(revision aef2c3eee9a0c73b2d215fe674c8e3eeb25b01a9)
+++ b/src/Main/Main.java	(date 1700755417818)
@@ -7,7 +7,14 @@
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
 
+      /*  // Inicializar instancias de las clases de la capa de datos
+        data.UserData userData = new data.UserData();
+        data.EntryData entryData = new data.EntryData();
 
+        // Inicializar instancias de las clases de la capa de negocio
+        business.UserManager userManager = new business.UserManager(userData);
+        business.EntryManager entryManager = new business.EntryManager(entryData);
+*/
         // Inicializar el manejador de menú
         presentation.MenuHandler menuHandler = new presentation.MenuHandler(scanner);
 
